samples/queries.js

```diff
@@ -111,12 +110,10 @@ function syncQuery(sqlQuery, projectId) {
     .catch(err => {
       console.error('ERROR:', err);
     });
-  // [END bigquery_sync_query]
 }
 
 function asyncQuery(sqlQuery, projectId) {
-  // [START bigquery_async_query]
-  // [START bigquery_build_client]
+  // [START bigquery_query]
```

Comment:
I assumed you'll be updating https://cloud.google.com/bigquery/docs/running-queries and https://cloud.google.com/bigquery/docs/reference/standard-sql/enabling-standard-sql with the new region tag?

Reply:
Yes